packages/dds/tree/src/test/simple-tree/discrepancies.spec.ts

```diff
@@ -0,0 +1,822 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import assert from "node:assert";
```

Comment:
the diff for this file is being lame. If the tests are similar to the old ones, it might be nice to put the file back in its old location for now so we can get a more useful diff.

If the contents are not mostly based on the old test suite, just let me know and I'd review this from scratch.

Reply:
I moved the file back temporarily